src/electos/ballotmaker/demo_data/ballot_lab_data.py

```diff
@@ -84,19 +84,67 @@ def candidate_name(candidate: Candidate):
 
 
 def candidate_party(candidate: Candidate, index):
-    """Get the name and abbreviation of the party of a candidate as it appears on a ballot."""
-    party = index.by_id(candidate.party_id)
-    name = text_content(party.name) if party else ""
-    abbreviation = (
-        text_content(party.abbreviation)
-        if party and party.abbreviation
-        else ""
-    )
-    result = {
-        "name": name,
-        "abbreviation": abbreviation,
-    }
-    return result
+    """Get the name and abbreviation of the party of a candidate as it appears on a ballot.
+
+       Drop either field from result if it isn't present.
+    """
+    # Note: party ID is returned to allow de-duplicating parties in callers.
+    id_ = candidate.party_id
```

Comment:
Here's my chance to be a pedantic Pythonista! I'm not going to let this chance pass me by...

The trailing underscore is (by convention) a way to avoid namespace collisions with Python keywords. Since "id" is not a Python keyword, there's no need for a trailing underscore.

```
>>> import keyword
>>> print(keyword.kwlist)
['False', 'None', 'True', 'and', 'as', 'assert', 'async', 'await', 'break', 'class', 'continue', 'def', 'del', 'elif', 'else', 'except', 'finally', 'for', 'from', 'global', 'if', 'import', 'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass', 'raise', 'return', 'try', 'while', 'with', 'yield']
>>> 
```

Reply:
> Since "id" is not a Python keyword, there's no need for a trailing underscore.

It's not a keyword but [`id` _is_ a builtin function](https://docs.python.org/3.8/library/functions.html#id) and it can be clobbered so that you can't use it in the scope of the variable. It's best to treat it as a keyword and not risk the collision.

Reply:
Per our conversation merging is deferred until after [milestone 1](https://github.com/TrustTheVote-Project/BallotLab/milestone/1)

Reply:
I stand corrected on `id_`. Well played! 

I think I'll use more descriptive names for my ids, like `candidate_id`, to avoid name collisions. But I'm now totally on board with `id_`